homework.md

```diff
@@ -4,101 +4,102 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```

Comment:
The question also changed with your latest update I guess, the original question was different. (multiplication signs).